### PR TITLE
 Factory for ZoneId

### PR DESCRIPTION
**Pull request by golem131**
Add types
Add creating some constants
Add factory for ZoneId
Add factory for java.time.Month
Add factory for java.time.Year

### CODE REVIEW

File: src/main/scala/com/markatta/timeforscala/package.scala [hunk 1]
```diff
@@ -50,7 +53,16 @@ package object timeforscala {
...
 
   type DateTimeFormatter = java.time.format.DateTimeFormatter
+  type IsoChronology = java.time.chrono.IsoChronology
+  type Chronology = java.time.chrono.Chronology
 
+  type TemporalAccessor = java.time.temporal.TemporalAccessor
+  type TemporalAmount = java.time.temporal.TemporalAmount
+  type ChronoUnit = java.time.temporal.ChronoUnit
+
+  type Date = java.util.Date
```
Comment (johanandren) [inline src/main/scala/com/markatta/timeforscala/package.scala]: This one I'm not so sure we want, it's mutable and generally horrible...

File: src/main/scala/com/markatta/timeforscala/richtimes.scala [hunk 2]
```diff
@@ -141,9 +138,9 @@ final class RichDuration private[timeforscala](d: Duration) extends Ordered[Dura
 
 final class RichPeriod private[timeforscala](p: Period) extends Ordered[Period] {
 
-  def days = p.getDays
-  def months = p.getMonths
-  def years = p.getYears
+  def days: Int = p.getDays
+  def months: Int = p.getMonths
+  def years: Int = p.getYears
```
Comment (johanandren) [inline src/main/scala/com/markatta/timeforscala/richtimes.scala:143]: 👍

**Review by johanandren — COMMENTED** [review]
LGTM except for pulling j.u.Date

**Comment by golem131** [comment]
@johanandren - fixed

**Comment by golem131** [comment]
@johanandren - please retry build, some strange problems on travis-ci

**Comment by johanandren** [comment]
It actually does not compile, see the travis output.

**Comment by golem131** [comment]
@johanandren - compilation ok now.

**Comment by johanandren** [comment]
Thanks!

**Pull request merged**